src/client/components/SentReviewsComponent/SentReviewsComponent.stories.js

```diff
@@ -10,6 +10,7 @@ export default { title: 'Sent Reviews Component' };
 const reviews = [
   {
     // Information for the person that gives the review
+    id: 123,
```

Comment:
Can I ask why is it a better idea to hardcode the id? In this way you are changing your data structure and it would have worked with passing index while mapping reviews and then have a key equal to that index? There´s no need for hard coding id. I think it would introduce more errors. Especially because my sum component followed your storybook data to render.

Reply:
because this is just for storybook, when using the actual data, it will use the id form those. And I think the id is unique, is already a value we already have in our database and will not change. Just think it is more appropriate than the index.

Reply:
There are ofc better sources, but you can look into [this](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)

Reply:
Okay.. its because I followed your storybook when making mine. Since my component is the sum component that lists your components. But I guess if it folllows the id in the database then its okay.. However me working with migrations and seeds earlier I learned that its not a good idea to hard code ids into our tables in database. But okay.. not sure who made the reviews table but I guess this is how it is now.. I guess I´ll make the changes accordingly. I can just see this making errors on many levels later on.

Reply:
Thanks.. for the link. I guess it is fine when we have multiple constants which I guess now we do since you have changed the structure of your data. Before how it was structured it wasn´t neccessary. Also if the id´s are not hard coded in the reviews table it would also not matter. But i guess I will make the changes then. Thank you

Reply:
Maybe you are getting a bit confused, the storybook is another thing, has no impact on our database. You are right, hardcoding values is surely a no 🙅. But storybook is just for testing/visualizing how a component will look like, it is more for UI purposes.

Reply:
No i know storybook and database are 2 different things but data we use is connected through rendering of the data. But sure, we can have completely different data in the database.. and get no errors. But maybe if in the rendering part we use some props connected to database then it can look a bit messy if our storybook is different. but idk. Ill just change it on my part.. its okay . Thank you for the chat and info :)